### PR TITLE
Kernel: Adjust includes after file reorganization in aarch64

### PR DESCRIPTION
**Pull request by tiaanl**
Changing the #include paths to where files were moved in a previous PR #13491.

### CODE REVIEW

File: Kernel/CMakeLists.txt [hunk 1]
```diff
@@ -411,27 +411,6 @@ else()
         ${PREKERNEL_SOURCES}
         ${RPI_SOURCES}
 
-        Arch/aarch64/BootPPMParser.cpp
```
Comment (linusg) [inline Kernel/CMakeLists.txt:414]: Why are we removing these?

Comment (tiaanl) [reply Kernel/CMakeLists.txt:414]: This was a mistake, I'll put these back.

**Review by linusg — CHANGES_REQUESTED** [review]
Please squash the commits as per the [contribution guidelines](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy).

File: Kernel/CMakeLists.txt (hunk 1 above)
Comment (tiaanl) [reply Kernel/CMakeLists.txt:414]: Some of those were duplicated coming from RPI_SOURCES and PREKERNEL_SOURCES

**Pull request merged**